test/functional/tests/cli/test_zero_metadata_command.py

```diff
@@ -0,0 +1,258 @@
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_zero_metadata_dirty_shutdown():
+    """
+        title: Test for '--zero-metadata' and dirty shutdown scenario.
+        description: |
+          Test for '--zero-metadata' with and without 'force' option on cache which had been dirty
+          shut down before.
+        pass_criteria:
+          - Zero metadata without force failed on cache after dirty shutdown.
+          - Zero metadata with force ran successfully on cache after dirty shutdown.
+          - Cache started successfully after operations.
+    """
+    with TestRun.step("Prepare cache and core devices."):
+        with TestRun.step("Prepare devices."):
+            cache_disk = TestRun.disks['cache']
+            cache_disk.create_partitions([Size(2, Unit.GibiByte)])
+            cache_dev = cache_disk.partitions[0]
+            core_disk = TestRun.disks['core']
+            core_disk.create_partitions([Size(5, Unit.GibiByte)])
+
+    with TestRun.step("Start cache."):
+        cache = casadm.start_cache(cache_dev, CacheMode.WT, force=True)
+        core = cache.add_core(core_disk)
+
+    with TestRun.step("Unplug cache device."):
+        cache_disk.unplug()
+
+    with TestRun.step("Stop cache without flush."):
+        try:
+            cache.stop(no_data_flush=True)
+        except CmdException:
+            TestRun.LOGGER.info("This could ended with error (expected)")
+
+    with TestRun.step("Plug cache device."):
+        cache_disk.plug()
+        time.sleep(1)
+
+    with TestRun.step("Start cache (expect to fail)."):
+        try:
+            cache = casadm.start_cache(cache_dev, CacheMode.WT)
+            TestRun.LOGGER.error("Starting cache should fail!")
+        except CmdException:
+            TestRun.LOGGER.info("Start cache failed as expected.")
+
+    with TestRun.step("Zero metadata on CAS device without force"):
+        output = TestRun.executor.run(cli.zero_metadata_cmd(str(cache_dev.path)))
+        if output.exit_code != 0:
+            cli_messages.check_stderr_msg(output, cli_messages.cache_dirty_shutdown)
+        else:
+            TestRun.LOGGER.error("Zeroing metadata without force should fail!")
+
+    with TestRun.step("Zero metadata on cache device with force"):
+        output = TestRun.executor.run(cli.zero_metadata_cmd(str(cache_dev.path), force=True))
+        if output.exit_code == 0:
+            TestRun.LOGGER.info("Zeroing metadata with force ran successfully!")
+        else:
+            TestRun.LOGGER.error("Zero metadata with force should works for cache device!")
+
+    with TestRun.step("Start cache."):
+        try:
+            cache = casadm.start_cache(cache_dev, CacheMode.WT)
+            TestRun.LOGGER.info("Cache started successfully.")
+        except CmdException:
+            TestRun.LOGGER.error("Start cache failed.")
+
+
+def prepare_devices():
+    cache_disk = TestRun.disks['cache']
+    cache_disk.create_partitions([Size(2, Unit.GibiByte)])
```

Comment:
Given the nature of the tests I think you could decrease the cache size to maybe 100MiB - that would save some execution time